truffle/contracts/Splitter.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity 0.5.10;
+
+import './Stoppable.sol';
+
+contract Splitter is Stoppable {
+    
+    struct Account {
+        uint    pendingWithdrawals;
+    }
+    
+    mapping(address => Account) accountStorage;
+
+    event LogEthSent(address sender, uint value, address receiver1, address receiver2);
+    event LogWithdrawed(address sender, uint amount);
+    
+    constructor() public payable {}
+    
+    function splitEth(address receiver1, address receiver2) public _onlyIfRunning payable returns(bool) {
+        require(msg.value <= msg.sender.balance, 'The sender balance shoud be higher the ether\'s value');
```

Comment:
Good idea to err on the side of caution. However, the protocol prevents someone from sending more Eth than they have so this check is an opportunity to save a little gas.